CHANGELOG.md

```diff
@@ -13,6 +14,14 @@
 - Asynchrone Alarme: Bei asynchroner Verarbeitung von schnell aufeinander folgenden Alarmen, wurde der Inhalt der Objekte typ, freq und data bereits vor dem Abschluss der Verarbeitung eines Alarms wieder überschrieben. Ergebnis hiervon war die Vermischung von RICs und Texten unterschiedlicher Alarme. Lösung über copy.deepcopy() [#394](https://github.com/Schrolli91/BOSWatch/issues/394)
 ##### Security
 
+### __[v2.#]__ - 05.09.2019
+##### Added
+##### Changed
+- POC-Decoder: Im POC-Text wird nach einem RegEx, welcher Koordinaten enthält, gesucht. Werden diese gefunden, so stehen zwei neu befüllte Data-Felder Lon bzw Lat zur Verfügung.
```

Comment:
Ich meinte eigentlich, dass dieser Eintrag in den schon vorhandenen, undatierten Block eingefügt werden sollte. Der von dir erstelle Changelog-Block fällt damit komplett raus. Einen neuen Block erstelle ich immer erst zum Release einer neuen Version

Reply:
Jetzt verstanden, erledigt